CHANGELOG.md

```diff
@@ -26,6 +27,9 @@ All notable changes to this project are documented in this file.
 - Fix typos in tsid balancing torque control config files (https://github.com/ami-iit/bipedal-locomotion-framework/pull/926)
 - Add option `FRAMEWORK_COMPILE_MotorCurrentTrackingApplication` in cmake file to install correctly the application motor-current-tracking (https://github.com/ami-iit/bipedal-locomotion-framework/pull/937)
 
+### Deprecated
+- Deprecate `FloatingBaseSystemKinematics` in favour of `FloatingBaseSystemVelocityKinematics` class in `ContinuousDynamicalSystem` component (https://github.com/ami-iit/bipedal-locomotion-framework/pull/950)
```

Comment:
Just for alignment: I am not sure if it make sense to call a rename "deprecation". Deprecation is typically used for something that is still there, but it should not used. If something is removed, no problem but to avoid confusion I would clarify that it has been renamed, not deprecated.